features/kubernetes-nmstate/README.md

```diff
@@ -80,4 +80,46 @@ To deploy kubernetes-nmstate use the `./deploy.sh` script in this folder.
 For testing you can use the `./testing.sh` script in this folder.
 The test will create a bond interface with the name `bond1` and attach `ens8` and `ens9` as the slave interfaces.
 
-*Note:* You can change the bond name and the slave interface names by editing the `myenv.file` file.
\ No newline at end of file
+*Note:* You can change the bond name and the slave interface names by editing the `myenv.file` file.
+
+### Node configuration examples
+
+```
+# IP configuration
```

Comment:
I'd say to move this out of the triple quotes and convert it to a list:

```
* IP configuration
```

Reply:
thanks, done.